pydocx/export/mixins/convert_root_upper_roman_list_to_heading.py

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from pydocx.openxml.packaging import NumberingDefinitionsPart
+
+
+class ConvertRootUpperRomanListToHeadingMixin(object):
+    HEADING_LEVEL = 'heading 2'
+
+    def _is_element_a_root_level_upper_roman_list_item(self, el):
```

Comment:
This method is surprisingly easy to read. The refactoring you've been working on is paying off in readability.